build.gradle

```diff
@@ -33,6 +33,7 @@ allprojects {
     repositories {
         google()
         jcenter()
+        mavenLocal()
```

Comment:
Does this need a TODO to be removed after testing?

Reply:
Added a TODO for this 👍